tests/modules/nf-core/deepvariant/test.yml

```diff
@@ -19,3 +19,12 @@
     - path: output/deepvariant/test_out.vcf.gz
       md5sum: 4c4b86a21ef77beb85d6fed80ce17c40
     - path: output/deepvariant/versions.yml
+
+- name: deepvariant test_deepvariant_no_fasta_gzi
+  command: nextflow run ./tests/modules/nf-core/deepvariant -entry test_deepvariant_no_fasta_gzi -c ./tests/config/nextflow.config
+  tags:
+    - deepvariant
+  exit_code: 1
+  stdout:
+    contains:
+      - "Failed to open FASTA index genome.fasta.gz.fai: No such file or directory"
```

Comment:
can you also add a test with the gzi?

Reply:
It looks like there is already one here: https://github.com/nf-core/test-datasets/tree/modules/data/pangenomics/homo_sapiens

Reply:
In this directory, no any bam or cram files based on this reference.

Reply:
Has the test been added? I don't see it 🙈

Reply:
@FriederikeHanssen 
Thanks for merging the test data, and the new tests added here now.
Could you please review it?

Reply:
@FriederikeHanssen Do you have time to look at this extra tests?
Thanks.